README.md

```diff
@@ -69,12 +69,53 @@ Standard Usage:
 - changes to **persistStore**:
   - 1. remove config argument (or replace with an empty object)
   - 2. remove all arguments from the callback. If you need state you can call `store.getState()`
-  - 3. All constants (ex: `{REHYDRATE, PURGE}`) has moved to `redux-persist/lib/constants` instead of `redux-persist/constants`
-- replace combineReducers with **persistCombineReducers**
+  - 3. All constants (ex: `REHYDRATE`, `PURGE`) has moved to `redux-persist/lib/constants` instead of `redux-persist/constants`
+- replace `combineReducers` with **persistCombineReducers**
   - e.g. `let reducer = persistCombineReducers(config, reducers)`
 - changes to **config**:
   - `key` is now required. Can be set to anything, e.g. 'primary'
   - `storage` is now required. For default storage: `import storage from 'redux-persist/lib/storage'`
+  
+```diff
+-import { REHYDRATE, PURGE } from 'redux-persist/constants'
++import { REHYDRATE, PURGE } from 'redux-persist/lib/constants'
+
+-import { combineReducers } from 'redux'
++import { persistCombineReducers } from 'redux-persist'
+
++import storage from 'redux-persist/lib/storage' // or whatever storage you are using
+
+ const config = {
++  key: 'primary',
++  storage
+ }
+
+-let reducer = combineReducers(reducers)
++let reducer = persistCombineReducers(config, reducers)
+
+ const store = createStore(
+   reducer,
+   undefined,
+   compose(
+     applyMiddleware(...),
+-    autoRehydrate()
+   )
+ )
+ 
+ const callback = ()
+
+ persistStore(
+   store, 
+-  config,
```

Comment:
this middle argument needs to exist actually in order for the callback to work. It can just be a empty object `{}`, although perhaps we should update the src so as it can be an optional?

Reply:
actually we should do this like redux, allow it to be null

Reply:
yeah, since you mention "1. remove config argument"

Reply:
this can now be null, but still needs to exist so these lines can be
```diff
-  config,
+ null,
```